cmd/kubeadm/app/componentconfigs/validation_test.go

```diff
@@ -286,7 +286,7 @@ func TestValidateKubeletConfiguration(t *testing.T) {
 				ComponentConfigs: kubeadm.ComponentConfigs{
 					Kubelet: &kubeletconfig.KubeletConfiguration{
 						CgroupsPerQOS:               true,
-						EnforceNodeAllocatable:      []string{"pods", "system-reserved", "kube-reserved"},
+						EnforceNodeAllocatable:      []string{"pods", "SystemReserved", "KubeReserved"},
```

Comment:
this introduces a modification in the accepted values for the kubelet v1beta1 config:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L709

arguably this should be noted in the comments above `EnforceNodeAllocatable`.
also raises the question if `SystemReserved`, `KubeReserved` are camel case, why is `none` and `pods` lowercase.